### PR TITLE
Fix readme for admin panel app

### PR DESCRIPTION
**Pull request by jarindr**

### CODE REVIEW

File: apps/admin_panel/README.md [hunk 1]
```diff
@@ -3,84 +3,4 @@
...
-  ├─ .gitignore       The list of files and folders that git should ignore
-  ├─ package.json     The manifest that contains project's metadata, dependencies, etc.
-  ├─ README.md        The document you are currently reading explaining how to get started
-  ├─ yarn.lock        **Autogenerated. Do not edit.** Yarn lockfile
-  └─ .babelrc         The babelrc config file
-```
-
-Jumping onto the project? Get started by reading `assets/package.json`, `assets/config/webpack.js`,
-then jump onto `assets/src/index.js`, the JS entrypoint of the project.
+If you wish to customize or start a development mode for admin panel, go here [Start developing admin panel](./assets/README.md)
```
Comment (unnawut) [inline apps/admin_panel/README.md]: To customize or start a development mode for the Admin Panel, see [Usage](./assets/README.md#usage).

File: apps/admin_panel/assets/README.md [hunk 2]
```diff
@@ -1,2 +1,28 @@
...
-A admin panel for e-wallet.
+# Admin Panel
+
+The OmiseGO Wallet admin panel for a provider to manage staff, users, roles and permissions,
+API access, tokens, transactions, etc.
+
+## Usage
+
+### Development
+
```
Comment (unnawut) [inline apps/admin_panel/assets/README.md]: The development mode allows you to develop the Admin Panel by running the Admin Panel locally, and letting it connect to an eWallet server elsewhere.

File: apps/admin_panel/README.md (hunk 1 above)
Comment (T-Dnzt) [reply apps/admin_panel/README.md]: I would actually say something simpler: 

To start the Admin Panel in development mode, follow these [instructions]().

File: apps/admin_panel/assets/README.md (hunk 2 above)
Comment (T-Dnzt) [reply apps/admin_panel/assets/README.md]: My 2 cents:

Running the Admin Panel in development mode locally lets you modify the code and see changes in real-time. You may connect to any eWallet as a backend, running locally or remotely.

**Pull request merged**